### DISCUSSION TITLE
Release 0.3.5 (1.20.0.22)

### DISCUSSION THREAD

**Discussion by rpoerschke** [Mojang/minecraft-editor#110] [Announcements]
_[Read the previous release notes here](https://github.com/Mojang/minecraft-editor/discussions/107)_ 

--- 

## Summary 

Another exciting week of updates to share with you all! We appreciate all the feedback you have been sharing with us!  Please continue to share your thoughts both here in GitHUb and on Discord! 

_Disclaimer:_ As our small team is heads-down working towards v0.4: The API Update, which will include documentation and a starter-kit, any bug and quality fixes week to week will be very modest in size and scope.  

--- 

## Fixed this release 

- Fixed a bug where the default command permission level of the client differed from what was displayed in a multiplayer session.  Client permissions are now reflected properly in the UI 

- Tool mode movement will now use the vanilla keyboard controls instead of hardcoded W/A/S/D/Space/Shift keys. Movement bindings can now be customized from the settings menu, but some keys might not be available if they are being used by other editor actions.  For now, updating the key bindings also affects the key bindings when playing Vanilla, but that will be updated in the future. 

 

--- 

## Next  

v0.4: The API Update will be focused on exposing the Editor APIs for you to begin building your own Editor extensions. For now we invite you to acquaint yourself with the existing [scripting](https://docs.microsoft.com/en-us/minecraft/creator/scriptapi/) efforts and get familiar with Typescript! 

  

Additionally, if you're new to the Editor GitHub this week, we encourage you to join the Bedrock Add-Ons Discord [#Editor channel](https://discord.gg/TZnQKFwGpt)! 

--- 

Thank you for being here! You're the best! ❤️

**Comment by SagarDulal09** [comment]
Minecraft is getting better in every update but i will like to suggest you to add some features and maps like left hane torch, dynamic lightening also map like sky block oneblock and hardcore. I kindly request you to add this feature in next update. This is my big request. So plz take my request serously. Thanks for this game. Hope to see above features in next update.
